### PR TITLE
TRACING-4460 |  Add Tempo AWS STS docs

### PR DESCRIPTION
**Pull request by pavolloffay**
Version(s):

4.12-4.17

This should be published as part of RHOSDT 3.3 release.

Issue:

https://issues.redhat.com/browse/TRACING-4460

Link to docs preview:

https://80044--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.html#aws-security-token-service-sts-installation

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/distr-tracing-tempo-storage-ref.adoc [hunk 1]
```diff
@@ -11,3 +11,70 @@ You can use the following configuration parameters when setting up a supported o
 --
 include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
 --
+
+=== AWS Security Token Service (STS) installation
+
+.Procedure
+
+. Create a custom AWS IAM Role associated with a trust relationship to Tempo's Kubernetes `ServiceAccount`:
```
Comment (IshwarKanse) [inline modules/distr-tracing-tempo-storage-ref.adoc:19]: Can we mention to create a S3 bucket in step 1.

**Comment by max-cx** [comment]
@pavolloffay, you're welcome to close this PR because I continue working on this content in https://github.com/openshift/openshift-docs/pull/80231.

**Pull request closed without merging**